### DISCUSSION TITLE
something wrong in es index init

### DISCUSSION THREAD

**Discussion by wushuangxiaoyu** [apache/skywalking#9977] [Q&A]
- skywalking9.2  && es6.8

get error message when init index
"as the final mapping would have more than 1 type"

![image](https://user-images.githubusercontent.com/26874082/202347669-2aee3d28-2bcd-4bdd-b6d3-639bf990acc6.png)

seem like es version question 

but it work normally in the local same  debug environment(9.2+6.8)

**Comment by kezhenxu94** [comment]
What's your exact es version? We've tested on 6.3.2 and it passed.

**Reply by wushuangxiaoyu** [reply]
thanks！
reason has been found
es has a common template mutexed with sw template
